modules/addNewBook.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] You have disabled Eslint checks on your code. Whiles your project looks great **please enable all linter checks** on your code to ensure that your code is robust code and clean.

Reply:
Very sorry for that. I enabled linters.